### DISCUSSION TITLE
"Customize" button - NetBeans installer in macOS

### DISCUSSION THREAD

**Discussion by piosol** [apache/netbeans#7434] [General]
Since version 19, the NetBeans installer on macOS does not have the "Customize" button for selecting the programming language. What happened to this button?

**Comment by neilcsmith-net** [comment]
It's not supported by NBPackage, the tool now used to generate the installer. There are a number of reasons why it doesn't, and why this wasn't a good option before. Including reducing complexity, ensuring that everything is correctly signed, and issues with ASF infrastructure if you then tried to enable a missing feature.

It's a feature that's unlikely to return. Why do you need it? Disk space?

**Reply by piosol** [reply]
I mean hardware resources (RAM, etc.). I only use NetBeans for PHP. NetBeans won't be "heavier" with all these packages?

**Reply by mhalachev** [reply]
You might try to deactivate unused features and observe the memory footprint before and after.

(Application Menu / Tools / Plugins / Installed [Java SE, C/C++,...] > Deactivate)

**Reply by neilcsmith-net** [reply]
Sure, although if you don't use them, they won't get activated in the first place.
